### ISSUE TITLE
[Site support request] playplus.tv

### ISSUE THREAD

**Issue by Rafa1310** [rg3/youtube-dl#18789]
## Please follow the guide below

- You will be asked some questions and requested to provide some information, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *issue* (like this: `[x]`)
- Use the *Preview* tab to see what your issue will actually look like

---

### Make sure you are using the *latest* version: run `youtube-dl --version` and ensure your version is *2019.01.02*. If it's not, read [this FAQ entry](https://github.com/rg3/youtube-dl/blob/master/README.md#how-do-i-update-youtube-dl) and update. Issues with outdated version will be rejected.
- [x] I've **verified** and **I assure** that I'm running youtube-dl **2019.01.02**

### Before submitting an *issue* make sure you have:
- [x] At least skimmed through the [README](https://github.com/rg3/youtube-dl/blob/master/README.md), **most notably** the [FAQ](https://github.com/rg3/youtube-dl#faq) and [BUGS](https://github.com/rg3/youtube-dl#bugs) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?type=Issues) the bugtracker for similar issues including closed ones
- [x] Checked that provided video/audio/playlist URLs (if any) are alive and playable in a browser

### What is the purpose of your *issue*?
- [ ] Bug report (encountered problems with youtube-dl)
- [x ] Site support request (request for adding support for a new site)
- [ ] Feature request (request for a new functionality)
- [ ] Question
- [ ] Other

---

### The following sections concretize particular purposed issues, you can erase any section (the contents between triple ---) not applicable to your *issue*

---

### If the purpose of this *issue* is a *bug report*, *site support request* or you are not completely sure provide the full verbose output as follows:

Add the `-v` flag to **your command line** you run youtube-dl with (`youtube-dl -v <your command line>`), copy the **whole** output and insert it here. It should look similar to one below (replace it with **your** log inserted between triple ```):

```
C:\Users\Suporte>youtube-dl -v https://www.playplus.tv/VOD/7572/db8d274a5163424e967f35a30ddafb8e -u myuser
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['-v', 'https://www.playplus.tv/VOD/7572/db8d274a5163424e967f35a30ddafb8e', '-u', 'PRIVATE']
Type account password and press [Return]:
[debug] Encodings: locale cp1252, fs mbcs, out cp850, pref cp1252
[debug] youtube-dl version 2019.01.02
[debug] Python version 3.4.4 (CPython) - Windows-10-10.0.17134
[debug] exe versions: ffmpeg N-92486-g8f875a90c4, ffprobe N-92486-g8f875a90c4
[debug] Proxy map: {}
[generic] db8d274a5163424e967f35a30ddafb8e: Requesting header
WARNING: Falling back on generic information extractor.
[generic] db8d274a5163424e967f35a30ddafb8e: Downloading webpage
[generic] db8d274a5163424e967f35a30ddafb8e: Extracting information
ERROR: Unsupported URL: https://www.playplus.tv/VOD/7572/db8d274a5163424e967f35a30ddafb8e
Traceback (most recent call last):
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmp5nai6j15\build\youtube_dl\YoutubeDL.py", line 793, in extract_info
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmp5nai6j15\build\youtube_dl\extractor\common.py", line 508, in extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmp5nai6j15\build\youtube_dl\extractor\generic.py", line 3320, in _real_extract
youtube_dl.utils.UnsupportedError: Unsupported URL: https://www.playplus.tv/VOD/7572/db8d274a5163424e967f35a30ddafb8e

```

---

### If the purpose of this *issue* is a *site support request* please provide all kinds of example URLs support for which should be included (replace following example URLs by **yours**):
- Single video: https://www.youtube.com/watch?v=BaW_jenozKc
- Single video: https://youtu.be/BaW_jenozKc
- Playlist: https://www.youtube.com/playlist?list=PL4lCao7KL_QFVb7Iudeipvc2BCavECqzc

Note that **youtube-dl does not support sites dedicated to [copyright infringement](https://github.com/rg3/youtube-dl#can-you-add-support-for-this-anime-video-site-or-site-which-shows-current-movies-for-free)**. In order for site support request to be accepted all provided example URLs should not violate any copyrights.

---

### Description of your *issue*, suggested solution and other information

Can you add this site to the list of supported sites?

**Comment by Rafa1310** [comment]
remitamine

Do you need my account? I put in the issue 'myuser' but my account is rafaelcosta1310@gmail.com

**Comment by remitamine** [comment]
> Do you need my account?

yes, an account is needed to access the video and add support for the website.

**Comment by Rafa1310** [comment]
> 
> 
> > Do you need my account?
> 
> yes, an account is needed to access the video and add support for the website.

Do you need login and password or just login?
My user is the email I quoted above, rafaelcosta1310@gmail.com

**Comment by Rafa1310** [comment]
> the username/email and password, as I said access to the video is needed to be able to add support for this website.

user: rafaelcosta_1310@hotmail.com
password: !qAG4p2*U2awgDRY

**Comment by Rafa1310** [comment]
The site for some reason no longer works for the domain playplus.tv, because when we access it now it redirects to playplus.com and with that youtube-dl can no longer perform downloads of the site. Can you fix this?

**Comment by Rafa1310** [comment]
When replacing .com by .tv in the link works, but will it always work that way?

**Comment by remitamine** [comment]
the new domain will be supported in the next version.

**Comment by Rafa1310** [comment]
When I download the file at best quality the best quality is not always the best quality. Example:

C: \ Users \ Support> youtube-dl https://www.playplus.tv/VOD/7572/1c3ced299654ecece2c134cf2de8a635 -u myuser -p mypassword -F
[PlayPlusTV] Downloading JSON metadata
[PlayPlusTV] Downloading JSON metadata
[PlayPlusTV] 1c3ced299654ecece2c134cf2de8a635: Downloading JSON metadata
[info] Available formats for 1c3ced299654ecece2c134cf2de8a635:
format code extension resolution note
0 mp4 426x240
1 mp4 640x360
2 mp4 854x480
3 mp4 1280x720
4 mp4 1920x1080
5 mp4 1920x1080 (best)

Sometimes the 4 format becomes the best quality instead of the 5 (best) format. So I have to test to see if 4 or 5 is the best quality available

**Comment by Rafa1310** [comment]
===================
Using the format 4
===================

C:\Users\Suporte\Desktop\Projetos>youtube-dl -v https://www.playplus.tv/VOD/7572/1c3ced299654ecece2c134cf2de8a635 -u myuser -p mypassword -f 4
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['-v', 'https://www.playplus.tv/VOD/7572/1c3ced299654ecece2c134cf2de8a635', '-u', 'PRIVATE', '-p', 'PRIVATE', '-f', '4']
[debug] Encodings: locale cp1252, fs mbcs, out cp850, pref cp1252
[debug] youtube-dl version 2019.01.10
[debug] Python version 3.4.4 (CPython) - Windows-10-10.0.17134
[debug] exe versions: ffmpeg N-92486-g8f875a90c4, ffprobe N-92486-g8f875a90c4
[debug] Proxy map: {}
[debug] Using fake IP 152.242.80.253 (BR) as X-Forwarded-For.
[PlayPlusTV] Downloading JSON metadata
[PlayPlusTV] Downloading JSON metadata
[PlayPlusTV] 1c3ced299654ecece2c134cf2de8a635: Downloading JSON metadata
[debug] Invoking downloader on 'http://gbbrpvd-sambavideos.akamaized.net/account/3170/3/2017-11-17/video/fe23873fef89ba196f23c884c1a87d96/ER7_MJ_FILME_2017-11-17_00-48-33_1080_23p_H.264_R7_PROXY.mp4'
[download] Destination: O Filme-1c3ced299654ecece2c134cf2de8a635.mp4
[download]   0.1% of 7.40GiB at  2.76MiB/s ETA 45:45

==========================
Using the format 5 (best)
==========================

C:\Users\Suporte\Desktop\Projetos>youtube-dl -v https://www.playplus.tv/VOD/7572/1c3ced299654ecece2c134cf2de8a635 -u myuser -p mypassword -f 5
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['-v', 'https://www.playplus.tv/VOD/7572/1c3ced299654ecece2c134cf2de8a635', '-u', 'PRIVATE', '-p', 'PRIVATE', '-f', '5']
[debug] Encodings: locale cp1252, fs mbcs, out cp850, pref cp1252
[debug] youtube-dl version 2019.01.10
[debug] Python version 3.4.4 (CPython) - Windows-10-10.0.17134
[debug] exe versions: ffmpeg N-92486-g8f875a90c4, ffprobe N-92486-g8f875a90c4
[debug] Proxy map: {}
[debug] Using fake IP 152.254.11.242 (BR) as X-Forwarded-For.
[PlayPlusTV] Downloading JSON metadata
[PlayPlusTV] Downloading JSON metadata
[PlayPlusTV] 1c3ced299654ecece2c134cf2de8a635: Downloading JSON metadata
[debug] Invoking downloader on 'http://gbbrpvd-sambavideos.akamaized.net/account/3170/3/2017-11-17/video/ebb71a3ed13c090bd7b6ab1c9af5e2c7/ebb71a3ed13c090bd7b6ab1c9af5e2c7_1080p.mp4'
[download] Resuming download at byte 4193280
[download] Destination: O Filme-1c3ced299654ecece2c134cf2de8a635.mp4
[download]   0.4% of 3.07GiB at  3.22MiB/s ETA 16:12
